app/World.php

```diff
@@ -168,6 +180,51 @@ public function __construct(int $id = 0, array $config = [])
                 $this->config['item.count.TwentyRupees2'] = 4;
         }
 
+        switch ($this->config('item.functionality')) {
+            case 'superexpert':
+                $this->config['rom.CapeMagicUsage.Normal'] = 0x01;
+                $this->config['rom.CapeMagicUsage.Half'] = 0x02;
+                $this->config['rom.CapeMagicUsage.Quarter'] = 0x02;
+                $this->config['rom.CaneOfByrnaInvulnerability'] = false;
+                $this->config['rom.PowderedSpriteFairyPrize'] = 0x79; // bees
+                $this->config['rom.BottleFill.Health'] = 0x00; // nothing
+                $this->config['rom.BottleFill.Magic'] = 0x00; // nothing
+                $this->config['rom.CatchableFairies'] = false;
+                $this->config['rom.CatchableBees'] = true;
+                $this->config['rom.StunItems'] = 0x00;
```

Comment:
Can we discuss just nix'ing the stun stopping? After a while of playing/testing this, it's just not fun or more challenging, but I could be wrong?

Reply:
I'm good with this, I think the boomerang/hookshot stun changes are the thing most people find objectionable with item functionality, I think removin git is a good call.  I'd still like to keep `rom.StunItems` as a config key.